<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPhoto</class>
 <widget class="QDialog" name="CPhoto">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>764</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>814</width>
    <height>764</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>CPhoto</string>
  </property>
  <widget class="PhotoView" name="ImgView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>771</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOpen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>720</y>
     <width>94</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string>LeftButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BtnExit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>720</y>
     <width>94</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string>RightButtons</string>
   </attribute>
  </widget>
  <widget class="QListWidget" name="ImgList">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>680</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnPlus">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>680</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string>RightButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BtnMinus">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>720</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string>RightButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BtnLeft">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>680</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string>RightButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BtnRight">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>680</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string>RightButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BtnNotes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>680</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string>LeftButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BtnDel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>680</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string>LeftButtons</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="BtnConfig">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>720</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnLibrary">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>680</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PhotoView</class>
   <extends>QGraphicsView</extends>
   <header>photoview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="LeftButtons"/>
  <buttongroup name="RightButtons"/>
 </buttongroups>
</ui>
